### ISSUE TITLE
Training VOC to cut the same object into two parts

### ISSUE THREAD

**Issue by lijain** [matterport/Mask_RCNN#2119]
During VOC data training, the same object in the graph will be cut into two parts. This can lead to poor results

**Comment by lijain** [comment]
The loss of training will not decrease in the later stage
276 240/240 [==============================] - 157s 653ms/step - loss: 2.0792 - rpn_class_loss: 0.0530 - rpn_bbox_loss: 0.6365 - mrcnn_class_loss: 0.2634 - mrcnn_bbox_loss: 0.5820 - mrcnn_mask_loss: 0.5443 - val_loss: 3.0459 - val_rpn_class_loss: 0.0556 - val_rpn_bbox_loss: 1.3190 - val_mrcnn_class_loss: 0.2207 - val_mrcnn_bbox_loss: 0.6704 - val_mrcnn_mask_loss: 0.7802
277 240/240 [==============================] - 157s 653ms/step - loss: 2.2502 - rpn_class_loss: 0.0713 - rpn_bbox_loss: 0.7473 - mrcnn_class_loss: 0.2890 - mrcnn_bbox_loss: 0.5904 - mrcnn_mask_loss: 0.5522 - val_loss: 3.0442 - val_rpn_class_loss: 0.0556 - val_rpn_bbox_loss: 1.3190 - val_mrcnn_class_loss: 0.2192 - val_mrcnn_bbox_loss: 0.6701 - val_mrcnn_mask_loss: 0.7802
278 240/240 [==============================] - 157s 654ms/step - loss: 2.0128 - rpn_class_loss: 0.0441 - rpn_bbox_loss: 0.5440 - mrcnn_class_loss: 0.2579 - mrcnn_bbox_loss: 0.5727 - mrcnn_mask_loss: 0.5942 - val_loss: 3.1218 - val_rpn_class_loss: 0.1314 - val_rpn_bbox_loss: 1.4366 - val_mrcnn_class_loss: 0.2470 - val_mrcnn_bbox_loss: 0.6365 - val_mrcnn_mask_loss: 0.6703
279 240/240 [==============================] - 157s 654ms/step - loss: 2.1681 - rpn_class_loss: 0.0598 - rpn_bbox_loss: 0.7417 - mrcnn_class_loss: 0.2622 - mrcnn_bbox_loss: 0.5500 - mrcnn_mask_loss: 0.5544 - val_loss: 3.7627 - val_rpn_class_loss: 0.2945 - val_rpn_bbox_loss: 1.9657 - val_mrcnn_class_loss: 0.3340 - val_mrcnn_bbox_loss: 0.6017 - val_mrcnn_mask_loss: 0.5669
280 240/240 [==============================] - 157s 653ms/step - loss: 2.4698 - rpn_class_loss: 0.0934 - rpn_bbox_loss: 0.9937 - mrcnn_class_loss: 0.2518 - mrcnn_bbox_loss: 0.5636 - mrcnn_mask_loss: 0.5672 - val_loss: 4.2504 - val_rpn_class_loss: 0.4537 - val_rpn_bbox_loss: 2.3967 - val_mrcnn_class_loss: 0.4146 - val_mrcnn_bbox_loss: 0.5607 - val_mrcnn_mask_loss: 0.4246
281 240/240 [==============================] - 157s 653ms/step - loss: 2.5130 - rpn_class_loss: 0.0947 - rpn_bbox_loss: 1.1045 - mrcnn_class_loss: 0.2611 - mrcnn_bbox_loss: 0.5286 - mrcnn_mask_loss: 0.5240 - val_loss: 4.2492 - val_rpn_class_loss: 0.4537 - val_rpn_bbox_loss: 2.3966 - val_mrcnn_class_loss: 0.4154 - val_mrcnn_bbox_loss: 0.5588 - val_mrcnn_mask_loss: 0.4246
282 240/240 [==============================] - 156s 651ms/step - loss: 2.1285 - rpn_class_loss: 0.0542 - rpn_bbox_loss: 0.6237 - mrcnn_class_loss: 0.3065 - mrcnn_bbox_loss: 0.5879 - mrcnn_mask_loss: 0.5561 - val_loss: 4.2464 - val_rpn_class_loss: 0.4537 - val_rpn_bbox_loss: 2.3966 - val_mrcnn_class_loss: 0.4082 - val_mrcnn_bbox_loss: 0.5632 - val_mrcnn_mask_loss: 0.4246
283 240/240 [==============================] - 156s 652ms/step - loss: 2.1794 - rpn_class_loss: 0.0599 - rpn_bbox_loss: 0.6778 - mrcnn_class_loss: 0.2551 - mrcnn_bbox_loss: 0.6019 - mrcnn_mask_loss: 0.5846 - val_loss: 3.1725 - val_rpn_class_loss: 0.2144 - val_rpn_bbox_loss: 1.4824 - val_mrcnn_class_loss: 0.3562 - val_mrcnn_bbox_loss: 0.5963 - val_mrcnn_mask_loss: 0.5231
284 240/240 [==============================] - 157s 655ms/step - loss: 1.9862 - rpn_class_loss: 0.0416 - rpn_bbox_loss: 0.5350 - mrcnn_class_loss: 0.2474 - mrcnn_bbox_loss: 0.5999 - mrcnn_mask_loss: 0.5622 - val_loss: 2.9984 - val_rpn_class_loss: 0.2142 - val_rpn_bbox_loss: 1.3485 - val_mrcnn_class_loss: 0.3901 - val_mrcnn_bbox_loss: 0.5801 - val_mrcnn_mask_loss: 0.4655
285 240/240 [==============================] - 157s 653ms/step - loss: 2.2251 - rpn_class_loss: 0.0544 - rpn_bbox_loss: 0.7730 - mrcnn_class_loss: 0.2379 - mrcnn_bbox_loss: 0.5808 - mrcnn_mask_loss: 0.5790 - val_loss: 2.1751 - val_rpn_class_loss: 0.0545 - val_rpn_bbox_loss: 0.6499 - val_mrcnn_class_loss: 0.3762 - val_mrcnn_bbox_loss: 0.6016 - val_mrcnn_mask_loss: 0.4929
286 240/240 [==============================] - 157s 655ms/step - loss: 2.3268 - rpn_class_loss: 0.0527 - rpn_bbox_loss: 0.8934 - mrcnn_class_loss: 0.2303 - mrcnn_bbox_loss: 0.5580 - mrcnn_mask_loss: 0.5923 - val_loss: 2.1653 - val_rpn_class_loss: 0.0545 - val_rpn_bbox_loss: 0.6499 - val_mrcnn_class_loss: 0.3722 - val_mrcnn_bbox_loss: 0.5958 - val_mrcnn_mask_loss: 0.4928
287 240/240 [==============================] - 157s 654ms/step - loss: 2.8561 - rpn_class_loss: 0.0702 - rpn_bbox_loss: 1.3611 - mrcnn_class_loss: 0.2565 - mrcnn_bbox_loss: 0.6033 - mrcnn_mask_loss: 0.5650 - val_loss: 2.1654 - val_rpn_class_loss: 0.0545 - val_rpn_bbox_loss: 0.6499 - val_mrcnn_class_loss: 0.3739 - val_mrcnn_bbox_loss: 0.5943 - val_mrcnn_mask_loss: 0.4928
288 240/240 [==============================] - 157s 654ms/step - loss: 2.3829 - rpn_class_loss: 0.0675 - rpn_bbox_loss: 0.9143 - mrcnn_class_loss: 0.2367 - mrcnn_bbox_loss: 0.5787 - mrcnn_mask_loss: 0.5856 - val_loss: 2.7441 - val_rpn_class_loss: 0.1349 - val_rpn_bbox_loss: 1.1308 - val_mrcnn_class_loss: 0.3284 - val_mrcnn_bbox_loss: 0.6442 - val_mrcnn_mask_loss: 0.5058
289 240/240 [==============================] - 156s 652ms/step - loss: 2.1430 - rpn_class_loss: 0.0667 - rpn_bbox_loss: 0.6668 - mrcnn_class_loss: 0.2647 - mrcnn_bbox_loss: 0.5731 - mrcnn_mask_loss: 0.5717 - val_loss: 2.4877 - val_rpn_class_loss: 0.0557 - val_rpn_bbox_loss: 0.9130 - val_mrcnn_class_loss: 0.2712 - val_mrcnn_bbox_loss: 0.7016 - val_mrcnn_mask_loss: 0.5461
290 240/240 [==============================] - 156s 651ms/step - loss: 2.5783 - rpn_class_loss: 0.1006 - rpn_bbox_loss: 1.0358 - mrcnn_class_loss: 0.2689 - mrcnn_bbox_loss: 0.5771 - mrcnn_mask_loss: 0.5960 - val_loss: 2.5681 - val_rpn_class_loss: 0.0560 - val_rpn_bbox_loss: 0.9788 - val_mrcnn_class_loss: 0.2412 - val_mrcnn_bbox_loss: 0.7329 - val_mrcnn_mask_loss: 0.5592
291 240/240 [==============================] - 157s 653ms/step - loss: 2.1756 - rpn_class_loss: 0.0663 - rpn_bbox_loss: 0.7220 - mrcnn_class_loss: 0.2368 - mrcnn_bbox_loss: 0.6068 - mrcnn_mask_loss: 0.5437 - val_loss: 2.5630 - val_rpn_class_loss: 0.0560 - val_rpn_bbox_loss: 0.9788 - val_mrcnn_class_loss: 0.2422 - val_mrcnn_bbox_loss: 0.7268 - val_mrcnn_mask_loss: 0.5592
292 240/240 [==============================] - 157s 656ms/step - loss: 2.7014 - rpn_class_loss: 0.0667 - rpn_bbox_loss: 1.2420 - mrcnn_class_loss: 0.2393 - mrcnn_bbox_loss: 0.5674 - mrcnn_mask_loss: 0.5860 - val_loss: 2.4912 - val_rpn_class_loss: 0.0557 - val_rpn_bbox_loss: 0.9129 - val_mrcnn_class_loss: 0.2732 - val_mrcnn_bbox_loss: 0.7036 - val_mrcnn_mask_loss: 0.5459
293 240/240 [==============================] - 157s 653ms/step - loss: 2.1281 - rpn_class_loss: 0.0581 - rpn_bbox_loss: 0.6004 - mrcnn_class_loss: 0.2687 - mrcnn_bbox_loss: 0.5807 - mrcnn_mask_loss: 0.6201 - val_loss: 2.4243 - val_rpn_class_loss: 0.0536 - val_rpn_bbox_loss: 0.7976 - val_mrcnn_class_loss: 0.3368 - val_mrcnn_bbox_loss: 0.6856 - val_mrcnn_mask_loss: 0.5507
294 240/240 [==============================] - 157s 654ms/step - loss: 2.2574 - rpn_class_loss: 0.0701 - rpn_bbox_loss: 0.7309 - mrcnn_class_loss: 0.2577 - mrcnn_bbox_loss: 0.5867 - mrcnn_mask_loss: 0.6120 - val_loss: 2.3816 - val_rpn_class_loss: 0.0528 - val_rpn_bbox_loss: 0.7372 - val_mrcnn_class_loss: 0.3721 - val_mrcnn_bbox_loss: 0.6717 - val_mrcnn_mask_loss: 0.5479
295 240/240 [==============================] - 156s 652ms/step - loss: 2.3200 - rpn_class_loss: 0.0596 - rpn_bbox_loss: 0.8652 - mrcnn_class_loss: 0.2797 - mrcnn_bbox_loss: 0.5799 - mrcnn_mask_loss: 0.5356 - val_loss: 2.3379 - val_rpn_class_loss: 0.0520 - val_rpn_bbox_loss: 0.6768 - val_mrcnn_class_loss: 0.4075 - val_mrcnn_bbox_loss: 0.6564 - val_mrcnn_mask_loss: 0.5451
296 240/240 [==============================] - 156s 652ms/step - loss: 1.9948 - rpn_class_loss: 0.0559 - rpn_bbox_loss: 0.5567 - mrcnn_class_loss: 0.2611 - mrcnn_bbox_loss: 0.5767 - mrcnn_mask_loss: 0.5444 - val_loss: 2.3330 - val_rpn_class_loss: 0.0520 - val_rpn_bbox_loss: 0.6768 - val_mrcnn_class_loss: 0.4052 - val_mrcnn_bbox_loss: 0.6521 - val_mrcnn_mask_loss: 0.5468
297 240/240 [==============================] - 157s 653ms/step - loss: 2.3321 - rpn_class_loss: 0.0701 - rpn_bbox_loss: 0.8996 - mrcnn_class_loss: 0.2674 - mrcnn_bbox_loss: 0.5667 - mrcnn_mask_loss: 0.5282 - val_loss: 2.3707 - val_rpn_class_loss: 0.0515 - val_rpn_bbox_loss: 0.7487 - val_mrcnn_class_loss: 0.3363 - val_mrcnn_bbox_loss: 0.6608 - val_mrcnn_mask_loss: 0.5734
298 240/240 [==============================] - 156s 651ms/step - loss: 2.1416 - rpn_class_loss: 0.0552 - rpn_bbox_loss: 0.6198 - mrcnn_class_loss: 0.2778 - mrcnn_bbox_loss: 0.5933 - mrcnn_mask_loss: 0.5955 - val_loss: 2.2969 - val_rpn_class_loss: 0.0479 - val_rpn_bbox_loss: 0.7111 - val_mrcnn_class_loss: 0.2884 - val_mrcnn_bbox_loss: 0.6300 - val_mrcnn_mask_loss: 0.6194
